src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Factory/EntityTypeFactory.php

```diff
@@ -123,7 +134,12 @@ private function buildOptions($environment)
     {
         $options = [];
         if ($environment === Constants::ENVIRONMENT_PRODUCTION) {
-            $options = ['validation_groups' => ['Default', 'production']];
+            $options = ['validation_groups' => ['Default', 'production'],
+                'attribute_service' => $this->attributeService
+            ];
+        }
+        if ($environment === Constants::ENVIRONMENT_TEST) {
+            $options['attribute_service'] = $this->attributeService;
```

Comment:
I don't think you need to inject the attribute_service this way into the type. But I might be wrong..